logprep/processor/domain_resolver/processor.py

```diff
@@ -54,7 +54,7 @@ class DomainResolverError(BaseException):
     """Base class for DomainResolver related exceptions."""
 
     def __init__(self, name: str, message: str):
-        super().__init__(f"DomainResolver ({name}): {message}")
+        super().__init__(f"DomainResolver ({name}): {message}")  # pragma: no cover
```

Comment:
Why are we ignoring this in the tests? Same as line 187. We could add maybe a test which mocks the thread_pool and has an UnicodeError as an 'side_effect'.

Reply:
I would prefer to delete it, because I have no clue, where it is raised in line 187

Reply:
deleted